### DISCUSSION TITLE
Fumadocs + Payload CMS

### DISCUSSION THREAD

**Discussion by pankajparashar** [fuma-nama/fumadocs#1010] [Q&A]
Has anyone tried to make Fumadocs work with Payload CMS? I'm going to take a jab at it over the weekend, but it'll be nice to know if anyone has tried already and what was the outcome?

**Comment by fuma-nama** [comment]
Haven't tried, but would be exciting if there's an example using Payload CMS, so we can add that to docs

**Comment by TinsFox** [comment]
I also want to try using strapi as cms. Writing with mdx is a bit difficult for those who are not familiar with it.

**Reply by fuma-nama** [reply]
You can check out our basehub and Sanity example, or try to integrate Fumadocs with Strapi or Payload CMS yourself

**Reply by TinsFox** [reply]
Thanks, I'll try it. BTW, Fumadocs is awesome!

**Comment by MFarabi619** [comment]
👋 Hello, I've shipped to production with Fumadocs + Payload CMS in [this](https://github.com/cuhacking/2025) project, with more in the pipeline. Opened #2053.
